libraries/AP_RCProtocol/AP_RCProtocol_SRXL.h

```diff
@@ -17,7 +17,10 @@
 
 #pragma once
 
-#include "AP_RCProtocol.h"
+#include "AP_RCProtocol_Backend.h"
+
+#if AP_RCPROTOCOL_SRXL_ENABLED
```

Comment:
Does anyone actually use SRXL? I would be tempted to switch it off by default

Reply:
I like that we'll have the option - but I reckon we can argue about that one later - maybe when we run out of space on some of our boards again :-)